ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -145,7 +149,7 @@ vlan_interfaces:
 {%                     if underlay_rfc5549 is arista.avd.defined(true) and overlay_mlag_rfc5549 is arista.avd.defined(true) %}
     ipv6_enable: true
 {%                     else %}
-    ip_address: {{ switch.mlag_l3_ip | arista.avd.default(switch.mlag_ip) }}/31
+    ip_address: {{ mlag_ibgp_peering_ip.ip | arista.avd.default(switch.mlag_l3_ip, switch.mlag_ip) }}/31
```

Comment:
`mlag_ibgp_peering_ip.ip ` will always have the same value (cf generated config)